dataland-frontend/src/components/resources/companyCockpit/FrameworkSummaryPanel.vue

```diff
@@ -84,6 +85,13 @@ onBeforeMount(() => {
       isUserUploader.value = result;
     })
     .catch((error) => console.log(error));
+  isUserDataOwnerForCompany(props.companyId, getKeycloakPromise)
+    .then((result) => {
+      if (!isUserUploader.value) {
+        isUserUploader.value = result;
```

Comment:
This seems wrong from a naming point of perspective.
The user is still not an uploader. But the user is a data-owner.

So let's rename the reactive variable `isUserUploader` to `isUserAllowedToUpload` or `isUserUploaderOrDataOwner` or introduce two seperate variables `isUserUploader` and `isUserDataOwner`.

I am open to other suggestions too.  But currently it is slightly misleading.

Reply:
Renamed to isUserAllowedToUpload